jablib/src/test/java/org/jabref/model/entry/AuthorTest.java

```diff
@@ -1,85 +1,66 @@
-        assertEquals("A. O.", Author.addDotIfAbbreviation("AO"));
-        assertEquals("A. O.", Author.addDotIfAbbreviation("AO."));
-        assertEquals("A. O.", Author.addDotIfAbbreviation("A.O."));
-        assertEquals("A.-O.", Author.addDotIfAbbreviation("A-O"));
-    }
-
-    @Test
-    void addDotIfAbbreviationDoesNotAddMultipleSpaces() {
-        assertEquals("A. O.", Author.addDotIfAbbreviation("A O"));
-    }
-
-    @ParameterizedTest
-    @ValueSource(strings = {"O.", "A. O.", "A.-O.",
-            "O. Moore", "A. O. Moore", "O. von Moore", "A.-O. Moore",
-            "Moore, O.", "Moore, O., Jr.", "Moore, A. O.", "Moore, A.-O.",
-            "MEmre", "{\\'{E}}douard", "J{\\\"o}rg", "Moore, O. and O. Moore",
-            "Moore, O. and O. Moore and Moore, O. O."})
-    void addDotIfAbbreviationDoNotAddDot(String input) {
-        assertEquals(input, Author.addDotIfAbbreviation(input));
-    }
-
-    @ParameterizedTest
-    @NullAndEmptySource
-    void addDotIfAbbreviationIfNameIsNullOrEmpty(String input) {
-        assertEquals(input, Author.addDotIfAbbreviation(input));
-    }
-
-    @ParameterizedTest
-    @ValueSource(strings = {"asdf", "a"})
-    void addDotIfAbbreviationLowerCaseLetters(String input) {
-        assertEquals(input, Author.addDotIfAbbreviation(input));
-    }
-
-    @Test
-    void addDotIfAbbreviationStartWithUpperCaseAndHyphen() {
-        assertEquals("A.-melia", Author.addDotIfAbbreviation("A-melia"));
-    }
-
-    @Test
-    void addDotIfAbbreviationEndsWithUpperCaseLetter() {
-        assertEquals("AmeliA", Author.addDotIfAbbreviation("AmeliA"));
-    }
-
-    @Test
-    void addDotIfAbbreviationEndsWithUpperCaseLetterSpaced() {
-        assertEquals("Ameli A.", Author.addDotIfAbbreviation("Ameli A"));
-    }
-
-    @Test
-    void addDotIfAbbreviationEndsWithWhiteSpaced() {
-        assertEquals("Ameli", Author.addDotIfAbbreviation("Ameli "));
-    }
-
-    @Test
-    void addDotIfAbbreviationEndsWithDoubleAbbreviation() {
-        assertEquals("Ameli A. A.", Author.addDotIfAbbreviation("Ameli AA"));
-    }
-
-    @Test
-    void bracesKept() {
-        assertEquals(Optional.of("{Company Name, LLC}"), new Author("", "", null, "{Company Name, LLC}", null).getFamilyName());
-    }
-
-    @ParameterizedTest
-    @ValueSource(strings = {"1", "1 23"})
-    void addDotIfAbbreviationIfStartsWithNumber(String input) {
-        assertEquals(input, Author.addDotIfAbbreviation(input));
-    }
+  @ParameterizedTest
+  @CsvSource({
+      "AO, 'A. O.'",
+      "AO., 'A. O.'",
+      "A.O., 'A. O.'",
+      "A-O, 'A.-O.'"
+  })
+  void addDotIfAbbreviationAddsDot(String input, String expected) {
+    assertEquals(expected, Author.addDotIfAbbreviation(input));
+  }
+
+  @ParameterizedTest
+  @CsvSource({
+      "'A O', 'A. O.'",
+      "'A-melia', 'A.-melia'",
+      "'AmeliA', 'AmeliA'",
+      "'Ameli A', 'Ameli A.'",
+      "'Ameli ', 'Ameli'",
+      "'Ameli AA', 'Ameli A. A.'"
```

Comment:
Please refactor using the knowledge provided at https://devdocs.jabref.org/code-howtos/testing.html#use-paramterizedtests